examples/sdk-app-example/src/main/java/com.bugsnag.android/example/ExampleActivity.kt

```diff
@@ -39,17 +40,11 @@ class ExampleActivity : AppCompatActivity() {
     }
 
     private fun performAdditionalBugsnagSetup() {
-        // Execute some code before every bugsnag notification
-        Bugsnag.beforeNotify { error ->
```

Comment:
can this be replaced with `.addOnError(fn)` or is that not implemented yet?

Reply:
I believe that hasn't been implemented yet.